Wikipedia/Code/SavedViewController.swift

```diff
@@ -314,6 +326,7 @@ extension SavedViewController: CollectionViewEditControllerNavigationDelegate {
     func didChangeEditingState(from oldEditingState: EditingState, to newEditingState: EditingState, rightBarButton: UIBarButtonItem?, leftBarButton: UIBarButtonItem?) {
         defer {
             navigationBar.updateNavigationItems()
+            ReadingListsFunnel.shared.logTappedEditButton()
```

Comment:
I'm not sure when all this method gets called, but I notice `edit_tab` is getting logged here even when I only tap the all articles or reading lists tab.

Reply:
Great catch, I left it at the wrong place; now it's only called when the `isEditing` flag is true